enterprise/server/hostedrunner/hostedrunner.go

```diff
@@ -274,7 +274,7 @@ func getExecProperty(execProps []*repb.Platform_Property, key string) string {
 	return ""
 }
 
-func (r *runnerService) withCredentials(ctx context.Context, req *rnpb.RunRequest) (context.Context, error) {
+func (r *runnerService) credentialOverrides(ctx context.Context, req *rnpb.RunRequest) ([]string, error) {
```

Comment:
nit: maybe call this `credentialEnvOverrides` to make it more clear what's being overridden?

```suggestion
func (r *runnerService) credentialEnvOverrides(ctx context.Context, req *rnpb.RunRequest) ([]string, error) {
```